### PR TITLE
Improve SystemOrganizer namings

### PR DESCRIPTION
**Pull request by jecisc**
Instead of using the vocabulary around "categroy", make it around "packageName" which is way more understandable.

### CODE REVIEW

**Comment by MarcusDenker** [comment]
```
Instance of SystemOrganizer did not understand #addCategory:

```

**Comment by jecisc** [comment]
I reverted a part of my last commit. I cannot find the last senders of this method easily. I'll do it later with the CI logs and my other cleanings

File: src/Monticello/MCOrganizationDefinition.class.st [hunk 1]
```diff
@@ -146,8 +146,6 @@ MCOrganizationDefinition >> summary [
 
 { #category : #unloading }
 MCOrganizationDefinition >> unload [
-	categories
-		do: [ :category | 
-			(SystemOrganization isEmptyCategoryNamed: category)
-				ifTrue: [ SystemOrganization removeCategory: category ] ]
+
+	categories do: [ :pacakgeName | (SystemOrganization classNamesInPackageNamed: pacakgeName) ifEmpty: [ SystemOrganization removePackageNamed: pacakgeName ] ]
```
Comment (privat) [inline src/Monticello/MCOrganizationDefinition.class.st]: pancakes?  noice!

**Review by privat — COMMENTED** [review]
With 23 commits, 50 files impacted and a big +584-573 diff, this PR is hard to review :(

File: src/Monticello/MCOrganizationDefinition.class.st (hunk 1 above)
Comment (jecisc) [reply src/Monticello/MCOrganizationDefinition.class.st]: I'll fix that :)

**Comment by jecisc** [comment]
I renamed only the methods in one class :(
When I look for stuff using the category naming convention, I have almost 6000 results. If I do PRs too small I'm afraid to still be doing PRs on this stuff in 5 years

But I'll try to do smaller still for the next ones!

**Comment by privat** [comment]
doing reformatting or mass renaming is independent commits can help.
Not one commit per file, but a single commit of mass renaming, then (or before) a single commit of reformatting, then additional commits if needed
the full diff will still be insane, but a commit-by-commit review will be a lot easier to spot weirdness or typo

**Comment by jecisc** [comment]
Marcus noticed that sometimes the categories are not packages but package tags.. I'll close it for now. We need more analysis

**Comment by MarcusDenker** [comment]
Categories are strings that have a package name as a prefix.

So it depends (I think) on which package exists already how to interpret a category like

Hello-Core-Something

It's a bit of a mess and needs some analysis of how to proceed.

**Pull request closed without merging**